src/main/java/Duke.java

```diff
@@ -1,10 +1,172 @@
+        getDateTime(); // due to testing purposes, will fail when coming timestamps
+        */
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addDeadline(ArrayList<Task> tasks, String taskDescription, int taskCounter) {
+        try {
+            String itemName = taskDescription.substring(LENGTH_DEADLINE);
+            String[] words = itemName.split("/");
+            Task newTask = new Deadline(words[0].trim(), words[1].trim());
+            tasks.add(newTask);
+            newTask.printAddDetails(taskCounter);
+        } catch (Exception e) {
+            System.out.println("Please input the date using the specified format");
+            taskCounter--;
+        }
+    }
+
+    public static void addEvent(ArrayList<Task> tasks, String taskDescription, int taskCounter) {
+        try {
+            String itemName = taskDescription.substring(LENGTH_EVENT);
+            String[] words = itemName.split("/");
+            Task newTask = new Event(words[0].trim(), words[1].trim());
+            tasks.add(newTask);
+            newTask.printAddDetails(taskCounter);
+        } catch (Exception e) {
+            System.out.println("Please input the date using the specified format");
+            taskCounter--;
+        }
+    }
+
+    public static void addTodo(ArrayList<Task> tasks, String taskDescription, int taskCounter) {
+        String itemName = taskDescription.substring(LENGTH_TODO);
+        Task newTask = new ToDo(itemName.trim());
+        tasks.add(newTask);
+        newTask.printAddDetails(taskCounter);
+    }
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        printWelcomeMessage();
+
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        int taskCounter = 0;
+
+        Scanner input = new Scanner(System.in);
+        String userCommand = input.nextLine();
+
+        while (!userCommand.equals("bye")){
+            String[] words = userCommand.split(" ");
+            int wordLength = words.length;
+            switch (words[0]) {
+            case "list":
+                printList(tasks);
+                break;
+            case "done":
+                if (wordLength != SIZE_DONE_COMMAND) {
+                    System.out.println("Wrong format for command \"done\"");
+                    break;
+                }
+                try {
+                    int index = Integer.parseInt(words[1]);
+                    completeTask(tasks, index);
+                } catch (Exception e) {
+                    System.out.println("Please input a valid number\n");
+                }
+                break;
```

Comment:
Inside the case "done": we can add a new function which extract these codes. It is not a coding standard violation, i am just commmenting to make your switch cases much cleaner.